packages/flutter/test/material/chip_theme_test.dart

```diff
@@ -36,6 +36,15 @@ DefaultTextStyle getLabelStyle(WidgetTester tester) {
   );
 }
 
+IconTheme getIconTheme(WidgetTester tester) {
+  return tester.widget<IconTheme>(
+    find.descendant(
+      of: find.byType(RawChip),
+      matching: find.byType(IconTheme),
+    ),
+  );
+}
```

Comment:
same as [`chip_test.dart`](https://github.com/flutter/flutter/blob/64f84f6a47d276e4d79cb10c203adbddeeeae336/packages/flutter/test/material/chip_test.dart#L37).
```suggestion
IconThemeData getIconData(WidgetTester tester) {
  final IconTheme iconTheme = tester.firstWidget(
    find.descendant(
      of: find.byType(RawChip),
      matching: find.byType(IconTheme),
    ),
  );
  return iconTheme.data;
}
```

Reply:
fixed